modules/apps/organizations/organizations-test/src/testIntegration/java/com/liferay/organizations/object/system/test/OrganizationSystemObjectDefinitionManagerTest.java

```diff
@@ -0,0 +1,301 @@
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.test.util.TestPropsValues;
+import com.liferay.portal.kernel.util.HashMapBuilder;
+import com.liferay.portal.kernel.util.LocaleUtil;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.test.rule.PermissionCheckerMethodTestRule;
+import com.liferay.portal.vulcan.util.LocalizedMapUtil;
+
+import java.util.List;
+import java.util.ListIterator;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Pedro Leite
+ */
+@Ignore
+@RunWith(Arquillian.class)
+public class OrganizationSystemObjectDefinitionManagerTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Before
+	public void setUp() throws Exception {
+		_organizationSystemObjectDefinitionManager =
+			_systemObjectDefinitionManagerRegistry.
+				getSystemObjectDefinitionManager("Organization");
+	}
+
+	@Test
+	public void testAddBaseModel() throws Exception {
+		_assertCount(0);
+
+		long organizationId1 = _addBaseModel(
+			HashMapBuilder.<String, Object>put(
+				"comments", RandomTestUtil.randomString()
+			).put(
+				"name", RandomTestUtil.randomString()
+			).build());
+
+		_assertCount(1);
+
+		long organizationId2 = _addBaseModel(
+			HashMapBuilder.<String, Object>put(
+				"comments", RandomTestUtil.randomString()
+			).put(
+				"name", RandomTestUtil.randomString()
+			).build());
+
+		_assertCount(2);
+
+		long organizationId3 = _addBaseModel(
+			HashMapBuilder.<String, Object>put(
+				"comments", RandomTestUtil.randomString()
+			).put(
+				"name", RandomTestUtil.randomString()
+			).build());
+
+		_assertCount(3);
+
+		long organizationId4 = _addBaseModel(
+			HashMapBuilder.<String, Object>put(
+				"comments", RandomTestUtil.randomString()
+			).put(
+				"name", RandomTestUtil.randomString()
+			).build());
+
+		_assertCount(4);
+
+		long organizationId5 = _addBaseModel(
+			HashMapBuilder.<String, Object>put(
+				"comments", RandomTestUtil.randomString()
+			).put(
+				"name", RandomTestUtil.randomString()
+			).build());
+
+		_assertCount(5);
+
+		_organizationLocalService.deleteOrganization(organizationId1);
+		_organizationLocalService.deleteOrganization(organizationId2);
+		_organizationLocalService.deleteOrganization(organizationId3);
+		_organizationLocalService.deleteOrganization(organizationId4);
+		_organizationLocalService.deleteOrganization(organizationId5);
+	}
+
+	@Test
+	public void testDeleteBaseModel() throws Exception {
+		long organizationId1 = _addBaseModel(
```

Comment:
The same thing here, I think we don't need to do the same thing 3 times, in my opinion just one is enough. Please see this example [NotificationQueueEntryLocalServiceTest#testDeleteNotificationQueueEntry](https://github.com/liferay/liferay-portal/blob/f50159380194ca7b2bcd4b8cb54c7492680ef912/modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/service/test/NotificationQueueEntryLocalServiceTest.java#L160)